docs/models/index.md

```diff
@@ -1,103 +1,27 @@
-       https://awf.ml.dev.web.auto/perception/models/label.txt
-
-
-# traffic_light_classifier
-
-$ mkdir -p ~/autoware_data/traffic_light_classifier/
-$ wget -P ~/autoware_data/traffic_light_classifier/ \
-       https://awf.ml.dev.web.auto/perception/models/traffic_light_classifier/v2/traffic_light_classifier_mobilenetv2_batch_1.onnx \
-       https://awf.ml.dev.web.auto/perception/models/traffic_light_classifier/v2/traffic_light_classifier_mobilenetv2_batch_4.onnx \
-       https://awf.ml.dev.web.auto/perception/models/traffic_light_classifier/v2/traffic_light_classifier_mobilenetv2_batch_6.onnx \
-       https://awf.ml.dev.web.auto/perception/models/traffic_light_classifier/v2/traffic_light_classifier_efficientNet_b1_batch_1.onnx \
-       https://awf.ml.dev.web.auto/perception/models/traffic_light_classifier/v2/traffic_light_classifier_efficientNet_b1_batch_4.onnx \
-       https://awf.ml.dev.web.auto/perception/models/traffic_light_classifier/v2/traffic_light_classifier_efficientNet_b1_batch_6.onnx \
-       https://awf.ml.dev.web.auto/perception/models/traffic_light_classifier/v2/lamp_labels.txt
-
-
-# traffic_light_fine_detector
-
-$ mkdir -p ~/autoware_data/traffic_light_fine_detector/
-$ wget -P ~/autoware_data/traffic_light_fine_detector/ \
-       https://awf.ml.dev.web.auto/perception/models/tlr_yolox_s/v2/tlr_yolox_s_batch_1.onnx \
-       https://awf.ml.dev.web.auto/perception/models/tlr_yolox_s/v2/tlr_yolox_s_batch_4.onnx \
-       https://awf.ml.dev.web.auto/perception/models/tlr_yolox_s/v2/tlr_yolox_s_batch_6.onnx \
-       https://awf.ml.dev.web.auto/perception/models/tlr_yolox_s/v2/tlr_labels.txt
-
-
-# traffic_light_ssd_fine_detector
-
-$ mkdir -p ~/autoware_data/traffic_light_ssd_fine_detector/
-$ wget -P ~/autoware_data/traffic_light_ssd_fine_detector/ \
-       https://awf.ml.dev.web.auto/perception/models/mb2-ssd-lite-tlr.onnx \
-       https://awf.ml.dev.web.auto/perception/models/voc_labels_tl.txt
-```
+ <https://awf.ml.dev.web.auto/perception/models/centerpoint/v2/pts_voxel_encoder_centerpoint.onnx> \
+ <https://awf.ml.dev.web.auto/perception/models/centerpoint/v2/pts_backbone_neck_head_centerpoint.onnx> \
+ <https://awf.ml.dev.web.auto/perception/models/centerpoint/v2/pts_voxel_encoder_centerpoint_tiny.onnx> \
+ <https://awf.ml.dev.web.auto/perception/models/centerpoint/v2/pts_backbone_neck_head_centerpoint_tiny.onnx> \
+ <https://awf.ml.dev.web.auto/perception/models/centerpoint/v2/centerpoint_ml_package.param.yaml> \
+ <https://awf.ml.dev.web.auto/perception/models/centerpoint/v2/centerpoint_tiny_ml_package.param.yaml> \
+ <https://awf.ml.dev.web.auto/perception/models/centerpoint/v2/centerpoint_sigma_ml_package.param.yaml> \
+ <https://awf.ml.dev.web.auto/perception/models/centerpoint/v2/detection_class_remapper.param.yaml> \
+ <https://awf.ml.dev.web.auto/perception/models/centerpoint/v2/deploy_metadata.yaml>
```

Comment:
Please add ```console ...

Reply:
@kminoda Thank you. I fixed at https://github.com/autowarefoundation/autoware-documentation/pull/589/commits/749739c742cfd37dab55071d32d23615bd2f6eaf

Reply:
Thanks!